pgmpy/tests/test_estimators/test_MaximumLikelihoodEstimator.py

```diff
@@ -8,8 +8,38 @@
 
 class TestMLE(unittest.TestCase):
     def setUp(self):
-        self.m1 = BayesianModel([('A', 'C'), ('B', 'C')])
-        self.d1 = pd.DataFrame(data={'A': [0, 0, 1], 'B': [0, 1, 0], 'C': [1, 1, 0]})
+        self.m1 = BayesianModel([('A','C'),('B','C')])
+        self.m2 = BayesianModel([('A','B'),('B','C')])
+        self.m3 = BayesianModel([('B','A'),('B','C')])
+        self.d1 = pd.DataFrame(data={'A':[0,0,1], 'B':[0,1,0], 'C':[1,1,0]})
+        self.d2 = pd.DataFrame(data={'A':[0,0,1,1], 'B':[0,1,0,1], 'C':[1,1,0,0]})
+
+
+    def test_get_parameters_causal(self):
+        mle = MaximumLikelihoodEstimator(self.m2, self.d2)
+        cpds = [ TabularCPD('A', 2, [[0.5],[0.5]]),
```

Comment:
@ankurankan @dmsedra The model is `A`→`B`→`C`, and in the data `A` is two times `0` and two times `1`, so the ML estimate is that `p(A=0)` is 50%, as in the code. The other cpds seem correct, too: `B` is one time `0` and one time `1` for `A=0`, and also for `A=1`. So `[[0.5, 0.5], [0.5, 0.5]]` is correct for `B`.